### DISCUSSION TITLE
I noticed that selecting file with @file is slow on a monorepo.

### DISCUSSION THREAD

**Discussion by muhajirdev** [yetone/avante.nvim#955] [General]
Just a typical javascript monorepo. 

I suspect it's because node_modules is not being ignored? So it scans everything?

**Comment by harrismcc** [comment]
Yeah, I'm seeing the same. things like `node_modules` or `.git` are not ignored, which slows things way down. Could be good to maybe use the .gitignore as a reference for what files to ignore?
